### ISSUE TITLE
9.2.5 Hook CreateFrame will crash the game

### ISSUE THREAD

**Issue by fang2hou** [Stanzilla/WoWUIBugs#232]
Test code:
```lua
hooksecurefunc("CreateFrame", function(name) print(name) end)
CreateFrame("Frame", nil, UIParent)
```

The error message:
![Snipaste_2022-03-29_16-33-50](https://user-images.githubusercontent.com/6694722/160559050-66204934-3e2d-4625-b60f-e44246f0bc7e.png)

**Comment by Nevcairiel** [comment]
Crashes are automatically reported to Blizzard, especially on the PTR. Make sure to submit a report with the re-production, that'll get to them faster then tracking it here.

**Comment by Meorawr** [comment]
Resolved in 9.2.5.43022.
